
% Load configs
\documentclass{CV}

\begin{document}

% ------ HEADER ------
\begin{center}
{\textbf{\LARGE Mirko LEDDA}}\\
Sharon Aviran Lab\\
Biomedical Engineering \& Genome Center\\
University of California, Davis, USA\\
Phone: \texttt{(510) 717-4889}\\
Email: \mail{maledda@ucdavis.edu}\\
Webpage:\href{https://mirkoledda.github.io}{~https://mirkoledda.github.io}\\
\end{center}
\medskip
\medskip
\begin{tabular}{p{0.15\textwidth} p{0.01\textwidth} p{0.80\textwidth}}
\tit{Objectives}
&& To obtain a 3-month industry internship in computational biology as part of the requirements of the Designated Emphasis in Biotechnology graduate program. Have 5 years of industry experience. Result oriented and thrives in large multinational teams and interdisciplinary research environments.
\end{tabular}

\medskip

% ------ SKILLS ------
\begin{tabular}{p{0.15\textwidth} p{0.85\textwidth}}
\tit{Skills}
& \begin{outline}
\vspace{-2em}
\1 \textbf{Computing:}
\2 Programming
\3 Python, R, Matlab (+ some C and Perl).
\3 Software development and release.
\3 Shell scripting and analysis on high-performance clusters.
\3 Common bioinformatics tools/pipelines.
\2 Data science
\3 Machine Learning.
\3 Statistics, algebra, calculus and probability theory.
\3 Algorithms and statistical methods development.
\3 Big data management and analysis.
\1 \textbf{Biology:}
\2 High-throughput sequencing (incl. library prep).
\2 Genomics, transcriptomics, metabolomics and GWAS
\2 Receptors biochemistry.
\2 Molecular, structural and cell biology.
\2 Bioprocesses and bioreactors.
\1 \textbf{Management/Business:}
\2 Project management and team building.
\2 Effective oral and written communication.
\2 IP process.
\2 Biological lab managment and safety (incl. MP, SOP and GLP).
\2 Teaching, consulting and mentoring.
\end{outline}
\end{tabular}
~


%% ------ AREAS OF SPECIALIZATION ------
%\section*{Areas of specialization}
%\centerline{Genetics - Genomics - Computational Biology - Structural Biology - Molecular Biology}
%\centerline{Bioinformatics - Statistics - Machine Learning - Software Development}
%~

% ------ EDUCATION ------
% \edu{year}{degree}{institution}
\section*{Education}
\textbf{Ph.D. in  Integrative Genetics and Genomics with Emphasis in Biotechnology (DEB*)}\\
University of California at Davis, CA, USA \hfill Expected graduation in Early 2019\\
~\\
\begin{tabular}{lcl}
    Thesis title&:&Hairpin in a haystack: structure-guided search for functional RNA elements\\
    Thesis advisor&:& Prof. Sharon Aviran, Biomedical Engineering \& Genome Center, UC Davis\\
    DEB advisor&:& Dr. Judith A. Kjelstrom, director, UC Davis Biotechnology Program\\
\end{tabular}\\
~\\
\begin{footnotesize}
*The \href{www.deb.ucdavis.edu}{DEB graduate program} is an inter-graduate group program involving faculty and students from 29 STEM disciplines. It is focused on providing cross-disciplinary training in critical areas of biomolecular research, promoting interdisciplinary team science, bioethics, entrepreneurship and professionalism as well as coordinating training in a biotechnology or life science company.
\end{footnotesize}\\
~\\
~
\textbf{B.Sc. in Life Sciences with Emphasis in Biotechnology} \hfill \years{2008}\\
University of Applied Sciences (HES-SO), Sion, Switzerland\\
~\\
\begin{tabular}{lcl}
    Topic&:& Strategies for the study of genes with unknown functions in \textit{Streptomyces}\\
    Advisors&:& Prof. Sergio Schmid, Institute of Life Technologies, University of Applied \\
    &&~~~~~~Sciences (HES-SO), Sion, Switzerland\\
    && Prof. Anna Maria Puglia, Dept. of Biological, Chemical and Pharmaceutical\\
    && ~~~~~~Sciences and Technologies, University of Palermo, Italy\\
\end{tabular}\\
~

% ------ SCIENTIFIC AND PROFESSIONAL EXPERIENCE ------
% \professional{year}{position}{institution}{description}
\section*{Research/Work experience}
\professional{2014-Current}{Ph.D. researcher}{UC Davis, CA}{Prof. Sharon Aviran}{
Computational and statistical methods development for nucleic acid structures. My contributions include:
\begin{outline}
\setlength\itemsep{0em}
\1 Characterized the statistical properties of RNA structure profiling data.
\1 Contributed to, and developed all the statistical bases and strategies for, \href{https://github.com/AviranLab/RNAprob}{\emph{RNAprob}}, a tool for data-directed RNA secondary structure prediction using probabilistic modeling.
\1 Conceived and developed \href{https://github.com/AviranLab/patteRNA}{\emph{patteRNA}}, a Machine Learning algorithm for the rapid mining of RNA secondary structure motifs from transcriptome-wide structure profiling data .
\1 Mentored new lab members and undergraduate summer students.
\1 Acting as lab manager and lab safety officer.
\1 Actively consulted in multiple research projects to tackle complex problems.
\end{outline}~
\vspace{-12pt}
}
\professional{2009-2014}{Research Assistant}{Nestl\'e Research Center, Lausanne, Switzerland}{Prof. Johannes le Coutre}{
Project and Lab manager for research on taste perception physiology. My contributions include:
\begin{outline}
%\begin{itemize}[label=\raisebox{0.25ex}{\tiny$\bullet$},leftmargin=*]
\setlength\itemsep{0em}
\1 Performed genome-wide association studies (GWAS) of human metabolism and taste perception;
\2 Discovered and patented new biomarkers of the health-status of the gastrointestinal tract.
\2 Developed statistical methods to analyze human taste phenotypic data.
\2 Discovered new genetic drivers of bitter taste perception, most notably for caffeine.
\1 Established the single cell Ca$^{2+}$-imaging technique in the lab and developed computational tools for automated image data analysis.
\1 Functionalized and \emph{in vitro} validated of several human, feline and rat GPCRs/TRP channels in mammalian cells (HEK, Hela, CHO, Chem-1 and primary rat DRG neurons).
\1 Performed receptor-interaction studies using siRNAs and co-expression approaches.
\1 Developed a method for the expression and purification of water-insoluble proteins in \emph{E. coli}.
\1 Managed research projects and activities. Produced deliverable on time.
\1 Mentored students and interns.
\1 Acted as Lab security officer, supervised supplies and ensured proper functioning of lab equipment.
\end{outline}~
\vspace{-12pt}
}
\professional{2008-2014}{Soldier specialist in biological weapons}{Swiss Army, Labor Spiez, Switzerland}{Dr. Christian Beuret}{
\vspace{-12pt}
\begin{outline}
\setlength\itemsep{0em}
\1 Development and validation of laboratory techniques for rapid identification of pathogenic bacteria, viruses and toxins.
\1 Team leader for research tasks.
\end{outline}~
\vspace{-20pt}
}

% ------ TEACHING ------
% \teaching{quarter + year}{position}{course name}{description}
\section*{Teaching experience}
\teaching{2018}{Guest Lecturer}{Quantitative Genetics and Selection Theory (PLS298), UC Davis}{Prof. Steve Knapp}{Graduate}{Gave a 1h30 lecture on fundamental concepts in Machine Learning.}{Very positive oral feedback from the students.}
\teaching{2017}{Lecturer}{Machine Learning Workshop for the Plant Sciences Dept., UC Davis}{Mirko Ledda}{Undergraduate, Graduate and Professors}{4h workshop on Machine Learning for the Ross-Ibarra, Knapp and Runcie Labs.}{Very positive oral feedback. Prof. Ross-Ibarra twitted about it \href{https://twitter.com/jrossibarra/status/940697831395831808}{here}.}
\teaching{2017}{Guest Lecturer}{Topics in BME: Computational Genomics (BIM189C), UC Davis}{Prof. Sharon Aviran}{Upper level undergraduate}{Two 2h lectures on fundamental concepts in Machine Learning.}{4.7/5 - 67\% excellent and 33\% very good ratings.}
\teaching{2016}{Teaching assistant}{Quantitative Genetics and Selection Theory (PLS298), UC Davis}{Prof. Steve Knapp}{Graduate}{Teaching R programming and the mathematical bases of selection theory in lab sessions.}{Very positive oral feedback from the students.}
\teachingnofeed{2015}{Course development}{Quantitative Genetics and Selection Theory (PLS298), UC Davis}{Prof. Steve Knapp}{Graduate}{Preparation of the teaching material for this newly proposed class.}

% ------ AWARDS ------
% \award{year}{name}{institution}{description}
\section*{Awards}
\award{2017}{UC Davis Graduate Student Travel Award}{UC Davis}{Competitive award to cover the cost to attend, as a speaker, the \emph{2017 [BC]2 Basel Computational Biology Conference} in Basel, Switzerland.}
\award{2016}{Registration Bursary}{Wellcome Genome Campus Scientific Conferences}{Competitive award to cover the cost to attend, as a speaker, the \emph{2016 Computational RNA Biology Conference} in Cambridge, UK.}
\award{2016}{Summer Graduate Student Researcher Award}{UC Davis}{3-months support for graduate research in engineering, computer science, and disciplines with engineering-related applications and methods.}

% MEMBERSHIPS ----
% \member{year}{name}{description}
\section*{Membership}
\member{2018}{The RNA Society}{Student member}


% ------ SCIENCE OUTREACH ------
% \outreach{year}{event}{location}{description}
\begin{tabular}{p{0.15\textwidth} p{0.85\textwidth}}
\tit{Community Service} 
& \outreach{2015-Current}{Graduate Student Association (GSA)}{UC Davis}{Representative for the IGG graduate program.}
& \outreach{2017}{IGG Annual Colloquium}{UC Davis}{Member of the organizing committee.}
& \outreach{2017}{Teen Biotech Challenge 2017}{DEB, UC Davis}{Judge for System and Computational Biology websites.}
& \outreach{2017}{Topics in Biomedical Engineering: Computational Genomics}{UC Davis}{Mentored three students for their final projects.}
& \outreach{2016}{Teen Biotech Challenge 2016}{DEB, UC Davis}{Judge for System and Computational Biology websites.}
& \outreach{2015}{Science in the Siskiyous}{Dunsmuir High School, Dunsmuir, CA, USA}{Presented biology research and taught basic genetic concepts to three $9^{\text{th}}\text{~to~}12^{\text{th}}$ grade high-school classes.}
& \outreach{2015}{Science vs Fiction}{Senior Center, Davis, CA, USA}{Presented common scientific misconceptions followed by an open discussion with seniors.}
& \outreach{2015}{IGG program}{UC Davis}{Mentor for all incoming international IGG students and mentor for a $1^{\text{st}}$ year IGG student.}
\end{tabular}
~

% ------ TALKS AND POSTERS ------
% \pubTalk{year}{authors}{title}{conference}{location}{date}{description}
\section*{Presentations and Posters}
\pubTalk{2017}{\textbf{Ledda M.} and Aviran S.}{patteRNA: Transcriptome-wide search for functional RNA elements via structural data signatures}{[BC]2 Basel Computational Biology Conference}{Congress Center, Basel, Switzerland}{September 13-$15^\text{th}$}{Speaker - 20min talk}
\pubTalk{2017}{\textbf{Ledda M.} and Aviran S.}{Transcriptome-wide search for functional RNA elements via structural data signatures}{Genome Research Day}{23andMe, Mountain View, CA}{June 1st}{Poster}
\pubTalk{2016}{\textbf{Ledda M.}, Deng F., Vaziri S., and Aviran S.}{Data-directed RNA secondary structure prediction using probabilistic modeling}{Computational RNA Biology Conference}{Wellcome Trust, Cambridge, UK}{October 17-$19^\text{th}$}{Speaker - 15min talk}

% ------ PUBLICATIONS ------
% \pubArticle{year}{authors}{title}{journal}{issue}{OPTIONAL doi}
\section*{Publications \footnotesize\text{( * indicates co-authorship)}}\medskip
% Published articles ------
\pubArticle{2018}{\textbf{Ledda M.} and Aviran S.}{patteRNA: transcriptome-wide search for functional RNA elements via structural data signatures}{Genome Biology}{in press}
\pubArticle{2016}{Choudhary K., Shih N.P., Deng F., \textbf{Ledda M.}, Li B. and Aviran S.}{Metrics for rapid quality control in RNA structure probing experiments}{Bioinformatics}{32(23): 2575-3583}{10.1093/bioinformatics/btw501}
\pubArticle{2016}{Deng F.*, \textbf{Ledda M.*}, Vaziri S. and Aviran S.}{Data-directed RNA secondary structure prediction using probabilistic modeling}{RNA}{22(8): 1109-19}{10.1261/rna.055756.115}
\pubArticle{2016}{Michlig González S., Meylan Merlini J., Beaumont M., \textbf{Ledda M.}, Tavenard A., Mukherjee R., Camacho S and le Coutre J.}{Acute Effects of single ingestion of TRPV1, TRPA1 and TRPM8 agonists on the energetic metabolism and the autonomic activity in healthy subjects}{Scientific Reports}{6: 20795}{10.1038/srep20795}
\pubArticle{2014}{Rueedi R.*, \textbf{Ledda M.*}, Nicholls A.W., Salek R.M., Marques-Vidal P., Morya E., Sameshima K., Montoliu I., Da Silva L., Collino S., Martin F-P., Rezzi S., Steinbeck C., Waterworth D.M., Waeber G., Vollenweider P., Beckmann J.S., le Coutre J., Mooser V., Bergmann S., Genick U.K., Kutalik Z.}{Genome-wide association study of metabolic traits reveals novel gene-metabolite-disease links}{PLoS Genetics}{10(2)}{10.1371/journal.pgen.1004132}
\pubArticle{2013}{\textbf{Ledda M.*}, Kutalik Z.*, Destito M.C.S., Souza M.M., Cirillo C. a., Zamboni A., Martin N., Morya E., Sameshima K., Beckmann J.S., le Coutre J., Bergmann S., Genick U.K.}{GWAS of human bitter taste perception identifies new loci and reveals additional complexity of bitter taste genetics}{Human Molecular Genetics}{23: 259-267}{10.1093/hmg/ddt404}
\pubArticle{2013}{Godinot N., Yasumatsu K., Barcos M.E., Pineau N., \textbf{Ledda M.}, Viton F., Ninomiya Y., le Coutre J. and Damak S.}{Activation of tongue-expressed GPR40 and GPR120 by non caloric agonists is not sufficient to drive preference in mice}{Neuroscience}{250: 20-30}{10.1016/j.neuroscience.2013.06.043}
\pubArticle{2013}{Montoliu I.*, Genick U.*, \textbf{Ledda M.}, Collino S., Martin F.P., Le Coutre J. and Rezzi S.}{Current status on genome-metabolome-wide associations: An opportunity in nutrition research}{Genes and Nutrition}{8: 19-27}{10.1007/s12263-012-0313-7}
\pubArticle{2011}{Genick U.K., Kutalik Z., \textbf{Ledda M.}, Souza Destito M.C., Souza M.M., Cirillo C. a., Godinot N., Martin N., Morya E., Sameshima K., Bergmann S., le Coutre J.}{Sensitivity of genome-wide-association signals to phenotyping strategy: The PROP-TAS2R38 taste association as a benchmark}{PLoS One}{6(11)}{10.1371/journal.pone.0027745}

% Submitted / In prep articles ------
% \pubRev{year}{authors}{title}{journal}
% \pubSub{year}{authors}{title}{journal}
% \pubPrep{year}{authors}{title}{journal}

% \subsection*{Manuscripts submitted / in-preparation}

% ------ PATENTS ------
% \pubPatent{year filed}{inventors}{title}{patent office}{patent #}{year issued}
\section*{Patents}
\pubPatent{2014}{Genick U.K., \textbf{Ledda M.}, Montoliu I., Le Coutre J., Rezzi S., Collino S., Martin F.P., Da Silva L.}{Genetic and urine-derived markers of human metabolic and gut microbial states}
\begin{tabular}{ll}
~~~~European Patent Office & \textit{EP2687845 A1} (issued in 2014)\\
~~~~US Patent Office & \textit{US Patent 20,150,160,191} (Issued in 2015)\medskip
\end{tabular}
\medskip
\vspace{1em}

% ------ LANGUAGES ------
\begin{tabular}{p{0.15\textwidth} p{0.85\textwidth}}
\tit{Hobbies} 
& Soccer, Alpine Ski and GoKart\\
& Hiking, traveling and taking (too) many pictures \\
& Building servers at home
\end{tabular}
~

\begin{center}
\Large \textbf{References upon request}
\end{center}

% ------ LANGUAGES ------
%\section*{Languages}
%\begin{multicols}{3}
%\begin{itemize}[label=\raisebox{0.25ex}{\tiny$\bullet$},leftmargin=*]
%\item French: Mother Tongue
%\item Italian: Mother Tongue
%\item English: Fluent
%\end{itemize}
%\end{multicols}
%~

% ------ COMPUTER SKILLS ------
%\section*{Programming Languages}
%\centerline{Python - R - Matlab - Perl - Bourne Shell}
%~
%\section*{Computer skills}
%\emph{Programming} \tab{Python - R - Matlab - Perl - Bourne Shell}\\
%\emph{Typesetting} \tab{LaTex - Markdown - Microsoft Office}\\

\end{document}